pkg/fuzzer/queue/retry.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2024 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package queue
+
+type retryer struct {
+	pq   *PlainQueue
+	base Source
+}
+
+// NewRetryer adds a layer that resends results with Status=Restarted.
+func NewRetryer(base Source) Source {
```

Comment:
Retry? Or what was the criteria to select Multiple over NewMultiplexer, but NewRetryer rather than Retry?